docs/development/machine-bootstrap.md

```diff
@@ -0,0 +1,22 @@
+# Machine-Controller integration into Gardener
+The Gardener deploys the [machine-controller-manager](https://github.com/gardener/machine-controller-manager) as part of a Shoot controlplane onto a Seed to manage the nodes/machines of the Shoot cluster. The machine-controller-manager will create, bootstrap, update and delete the nodes of the Shoot cluster and the underlying virtual machines.
+
+## Flow of creating and bootstrapping Machines/Nodes
+The following steps will be ensured by the garden-controller-manager during a creation/reconciliation flow:
+
+- The machine-controller-manager gets deployed into the Shoot namespace of the Seed.
+- The proper ``<cloudprovider>MachineClass`` objects and their corresponding secrets gets deployed in the Shoot namespace of the Seed. Those secrets contain information how to bootstrap the nodes and will be used by the machine-controller-manager to configure cloud-config/user-data of the VM.
```

Comment:
s/gets/get

Reply:
Let's also mention that the secret contains the credentials of the cloud provider account where to create the VMs.